### PR TITLE
Attach guests to events

### PR DESCRIPTION
**Pull request by csrubin**

### CODE REVIEW

**Comment by csrubin** [comment]
UI changes were cursory, mostly used for testing.

**Comment by csrubin** [comment]
Update functionality tested via API call, not available in UI. See TAS-125

File: how2meet/db/models.py [hunk 1]
```diff
@@ -28,23 +28,37 @@ class Event(Base):
...
 
-class Invite(Base):
-    __tablename__ = "invites"
+class Guest(Base):
+    __tablename__ = "guests"
 
     id = Column(String(32), primary_key=True, index=True)
     name = Column(String(100))
-    email = Column(String(100), nullable=True)
+    email = Column(String(100), nullable=True)  # TODO: how to require at least one (email OR phone)? Maybe not what we want anyway
```
Comment (BenMemberg) [inline how2meet/db/models.py:42]: I don't know that there's value enforcing this at the model/db level? Seems like something we can just require in the frontend when adding a guest.

Comment (BenMemberg) [reply how2meet/db/models.py:42]: Actually thinking about this more... I don't agree that a guest should require a contact method at all. If we want to follow When2Meet's example of frictionless users, we should allow for a guest to RSVP with no connecting info.

File: how2meet/db/models.py [hunk 2]
```diff
@@ -28,23 +28,37 @@ class Event(Base):
...
-    verified = Column(Boolean, nullable=True)
-    event_id = Column(Integer, ForeignKey("events.id"), nullable=False)
+    event_id = Column(String(32), ForeignKey("events.id"), nullable=False)
 
     def __repr__(self):
-        return f"Invite(name='{self.name}', contact='{self.email if self.email else self.phone}')"
+        return f"Guest(name='{self.name}', contact='{self.email if self.email else self.phone}')"
+
+
+# class Invite(Base):
```
Comment (BenMemberg) [inline how2meet/db/models.py]: Probably don't need this anymore 😂

File: how2meet/db/schemas.py [hunk 3]
```diff
@@ -44,19 +44,44 @@ class EventUpdate(BaseModel):
...
+class GuestUpdate(Guest):
+    id: Optional[str] = Field(default=None, description="The ID of the guest")
+    name: Optional[str] = Field(default=None, description="The name of the guest")
+    email: Optional[str] = Field(default=None, description="The email of the guest")
+    phone: Optional[int] = Field(default=None, description="The phone number of the guest")
+    status: Optional[str] = Field(default=None, description="The status of the guest")
+    event_id: Optional[str] = Field(default=None, description="The event ID of the guest")
+
+
+# class Invite(BaseModel):
```
Comment (BenMemberg) [inline how2meet/db/schemas.py]: Also remove these please.

File: how2meet/db/schemas.py [hunk 4]
```diff
@@ -31,7 +31,7 @@ class EventCreate(Event):
     pass
 
 
-class EventUpdate(BaseModel):
+class EventUpdate(Event):
```
Comment (BenMemberg) [inline how2meet/db/schemas.py:34]: I'm a little confused by the consequences of overwriting all of the original class fields with these pydantic `Field` instances. I guess if you tested it and it didn't error then it's not a blocker for now.

File: how2meet/main.py [hunk 5]
```diff
@@ -22,12 +22,12 @@
     redoc_url="/v1/redoc",
 )
 api_app.include_router(events.router)
-api_app.include_router(invites.router)
+# api_app.include_router(invites.router)
```
Comment (BenMemberg) [inline how2meet/main.py]: Remove comment

**Comment by csrubin** [comment]
Screenshot 2024-02-12 at 1 30 59 PM
et/assets/19996600/5082f274-90a9-48df-9996-b2a89e44ceae">

**Comment by csrubin** [comment]
Screenshot 2024-02-12 at 1 30 59 PM
Screenshot 2024-02-12 at 1 31 04 PM

**Comment by csrubin** [comment]
Screenshot 2024-02-12 at 1 32 17 PM
Screenshot 2024-02-12 at 1 32 26 PM

**Review by BenMemberg — APPROVED** [review]

**Pull request merged**